Dockerfile

```diff
@@ -10,6 +10,8 @@ RUN pip install deepdiff==1.7.0
 RUN pip install pytest-cov==3.0.0
 RUN pip install --upgrade rdflib==5.0.0
 RUN pip install -e git+https://github.com/hydroshare/hsmodels.git@0.5.3#egg=hsmodels
+RUN pip install aiohttp==3.8.3
+RUN pip install PyLD
```

Comment:
you're right, I should've. I'm not concerned because we have it pinned in hs_docker_base
https://github.com/hydroshare/hs_docker_base/blob/develop/requirements.txt#L130